axelrod/result_set.py

```diff
@@ -1,9 +1,14 @@
 from collections import namedtuple, Counter
+from multiprocessing import cpu_count
 import csv
+import itertools
 
-from numpy import mean, nanmedian, std
+from numpy import mean, nanmedian, std, array, nan_to_num
```

Comment:
minor: I'd prefer to `import numpy as np` and then use `np.mean` and such because names like mean, std, array are common and there could be collisions.

Reply:
Yeah I agree, I'll change this.